### DISCUSSION TITLE
JS regex advice

### DISCUSSION THREAD

**Discussion by GCAT01** [tutao/tutanota#7253] [Q&A]
I am trying to create a filter rule to move emails that come in to my "orders" address to a folder if and only if they have certain words in the subject field. For example:
Subject: Here are your shipping details
To: myorders@tutamail

JS regex: /subject:(?=.*ship|.*order|.*invoice)(?=to:.*myorders@tutamail)/gmis

However, I am running into a problem as adding the ".*" in the subject lookahead is matching anything and everything in the header. How can I make this work?

**Comment by charlag** [comment]
this works I think:
```regex
subject: .*(ship|order|details).*\nto: myorders@tutamail
```
https://regex101.com/r/8UzWDV/1

**Reply by charlag** [reply]
missing one match before email address but you got the idea

**Comment by GCAT01** [comment]
Honestly, I don't know what the problem was but without changing a thing it is working beautifully. I can order things and get the order-related emails in my inbox but all of the other garbage I'm sent (the spam you get once companies have your email) just gets trashed.
